x-pack/plugins/ml/public/application/model_management/test_models/models/inference_base.ts

```diff
@@ -81,8 +82,10 @@ export abstract class InferenceBase<TInferResponse> {
     protected readonly trainedModelsApi: ReturnType<typeof trainedModelsApiProvider>,
     protected readonly model: estypes.MlTrainedModelConfig,
     protected readonly inputType: INPUT_TYPE,
-    protected readonly deploymentId: string
+    protected readonly deploymentId: string,
+    protected readonly pipeline?: estypes.IngestPipeline
```

Comment:
`pipeline` isn't needed here as we can just set the private `pipelineForCreation` variable from the constructor, like the others.

```suggestion
    private readonly pipelineForCreation?: estypes.IngestPipeline
```

Reply:
Good catch - updated in 869c46cd1958becccee926172373b5e52d125d87